Shaders/compositor_pass/compositor_pass.frag.glsl

```diff
@@ -240,6 +245,19 @@ void main() {
 	}
 #endif
 
+#ifdef _CDistort
+	// const float compoDistortStrength = 4.0;
+	#ifdef _CPostprocess
+		float uStrength = PPComp14;
+	#else
+		float uStrength = compoDistortStrength;
+	#endif
+	float uX = time * uStrength;
+	texCo.y = texCo.y + (sin(texCo.x*4.0+uX*2.0)*0.01);
+	texCo.x = texCo.x + (cos(texCo.y*4.0+uX*2.0)*0.01);
+	fragColor.rgb = texture(tex, texCo).rgb;
```

Comment:
Do we really need this line of code? The shader reads from `tex` later in the code anyways, and because you modify `texCo` this should affect the other texture writes in the same way.

Reply:
Removed the line, thanks. 👍Initially I thought when you suggested removing it in the Discord thread that it was an "optional" removal, my mistake.